src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourtEvent.kt

```diff
@@ -54,14 +54,14 @@ class CourtEvent(
     value = [
       JoinColumnOrFormula(
         formula = JoinFormula(
-          value = "'" + CourtEventType.MOVE_RSN + "'",
+          value = "'" + MovementReason.MOVE_RSN + "'",
           referencedColumnName = "domain",
         ),
       ),
       JoinColumnOrFormula(column = JoinColumn(name = "COURT_EVENT_TYPE", referencedColumnName = "code")),
     ],
   )
-  val courtEventType: CourtEventType,
+  val courtEventType: MovementReason,
 
```

Comment:
This actually uses movement reasons, since we can only have one class by reference domain switch to MovementReason - this is same for prison-api